styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -379,4 +380,9 @@ private static boolean isActiveState(RunStateData runStateData) {
 
     return Stream.concat(processedStates.stream(), waitingStates.stream()).collect(toList());
   }
+
+  private static List<Instant> instants(Instant start, Instant end, Schedule schedule) {
+    return start.isAfter(end) ? instantsInReversedRange(start, end, schedule) :
```

Comment:
In order to always return the same set of instants, maybe this should be:

```
return start.isAfter(end))
  ? Lists.reverse(instantsInRange(end, start, schedule))
  : instantsInRange(start, end, schedule);
```

If not, at least please format the ternary conditional operator like this (when it does not fit on one line):
```
return start.isAfter(end) 
  ? instantsInReversedRange(start, end, schedule) 
  : instantsInRange(start, end, schedule);
```

Reply:
It will be different because of inclusive and exclusive. Will fix the format.